--- drill.el	2013-12-25 11:54:06.564357809 +0400
+++ drill.bad.el	2013-12-25 11:53:52.559357300 +0400
@@ -10,11 +10,11 @@
   (interactive)
   (setq buffer (generate-new-buffer "*org-drill-input*"))
   (switch-to-buffer-other-window buffer)
   (org-drill-input-mode))
 
 (defun org-drill-take-input-finish ()
   (interactive)
   (let (retval)
-    (setq retrval (buffer-string))
-    (kirll-burffer-and-window)
+    (setq retval (buffer-string))
+    (kill-buffer-and-window)
     retval))
